spec/seeds_init.sql

```diff
@@ -0,0 +1,21 @@
+CREATE TABLE customers (
+    id SERIAL,
+    customer_name TEXT
+);
+
+CREATE TABLE items (
+    id SERIAL,
+    item_name TEXT,
+    item_price INT,
+    item_quantity INT
+);
+
+CREATE TABLE orders (
```

Comment:
Potentially more than is required here by using the 'TIMESTAMP' data type. Would date have sufficed here?